From 2b4e1c365ed3f77951304ce6222dc1b78789301d Mon Sep 17 00:00:00 2001
From: renchenglei <chenglei.ren@intel.com>
Date: Thu, 9 Mar 2023 13:34:39 +0800
Subject: [PATCH] INTERNAL: Fix Coverity issue

This is to help fix Coverity issue in mesa
project.

Tracked-On: OAM-105550
Signed-off-by: Ren Chenglei <chenglei.ren@intel.com>
---
 src/egl/drivers/dri2/platform_android.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/egl/drivers/dri2/platform_android.c b/src/egl/drivers/dri2/platform_android.c
index 2d494717f55..537827a11bf 100644
--- a/src/egl/drivers/dri2/platform_android.c
+++ b/src/egl/drivers/dri2/platform_android.c
@@ -254,7 +254,7 @@ static int get_ycbcr_from_flexlayout(struct android_flex_layout *outFlexLayout,
              ycbcr->chroma_step = outFlexLayout->planes[i].h_increment;
          break;
          default:
-             _eglLog(_EGL_WARNING,"unknown component 0x%x", __func__, outFlexLayout->planes[i].component);
+             _eglLog(_EGL_WARNING,"%s: unknown component 0x%x", __func__, outFlexLayout->planes[i].component);
          break;
        }
   }
-- 
2.40.1

